src/engine/osaka.md

```diff
@@ -47,6 +52,10 @@ The fields are encoded as follows:
 
 `proofs` **MUST** contain exactly `CELLS_PER_EXT_BLOB` cell proofs.
 
+### BlobAndProofV3
+
+Alias for `BlobAndProofV2`.
```

Comment:
Couldn’t we just use the same `BlobAndProofV2`? Method and structure versions shouldn’t necessarily be aligned

Reply:
We can! I just tried to follow my understanding of the convention, which seemed to copy the version from the method into the data structure (because otherwise BlobAndProof would be V1, even if it was introduced with GetBlobsV2).

Reply:
Dropped the alias here: https://github.com/ethereum/execution-apis/pull/674/commits/36f2d3e1165ee4a7907029c5b92eabdaa561b96e

If folks want it back, we can revert that commit.